gtk4/Gir.toml

```diff
@@ -335,6 +335,9 @@ trust_return_value_nullability = false
     [[object.function]]
     name = "accelerator_valid"
     manual = true # to make use of gdk::keys::Key
+    [[object.function]]
+    name = "disable_setlocale"
+    manual = true # to use assert_not_initialized
```

Comment:
```suggestion
    assertions = "skip"
```
should be enough, no need to make it a `manual` function

Reply:
That'd skip the entire assert though - the manual function adds an assert that it's called before gtk::init

Reply:
see https://github.com/gtk-rs/gir/pull/1064

Reply:
Thanks, pushed a new update